app/search_builders/curation_concerns/README.md

```diff
@@ -0,0 +1,69 @@
+# Search Builders
+
+Building searches is core to any Blacklight app, and CurationConcerns is no exception.  
+This directory contains our Search Builders, so named because the initial design followed a builder pattern, meaning
+that when invoked to set values, methods return the object itself, so that invocations can be chained, like:
+
+```ruby
+builder = Blacklight::SearchBuilder.new(processor_chain, scope)
+            .rows(20)
+            .page(3)
+            .with(q: 'Abraham Lincoln')
+```
+
+However, at this level, many if not most of the additional methods do not follow this pattern.  
```

Comment:
Most of the methods in the Blacklight::SearchBuilder itself do follow the pattern.  But we get a descendent of that class w/ many other behaviors and modules mixed in.  That's the difference.